src/Semoda/Semoda/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -18,6 +21,18 @@ public static void AddAppServices(this IServiceCollection collection)
             collection.AddSingleton<MainWindowViewModel>();
             collection.AddSingleton<DashboardPageViewModel>();
             collection.AddSingleton<SettingsPageViewModel>();
+
+            {
+                // Load settings from JSON and add for dependency injection
+                var configuration = new ConfigurationBuilder()
+                   .SetBasePath(Directory.GetCurrentDirectory())
```

Comment:
We should move the configuration path to a different location. To make it less easy to manipulate the file. Maybe the ApplicationData is a good location?